demos/continuous_batching/agentic_ai/README.md

```diff
@@ -0,0 +1,143 @@
+mkdir models
+```
+Run `export_model.py` script to download and quantize the model:
+
+> **Note:** The users in China need to set environment variable HF_ENDPOINT="https://hf-mirror.com" before running the export script to connect to the HF Hub.
+
+::::{tab-set}
+
+:::{tab-item} CPU
+```console
+python export_model.py text_generation --source_model Qwen/Qwen3-8B --weight-format int8 --config_file_path models/config.json --model_repository_path models --tools_model_type qwen3 --overwrite_models
+```
+:::
+
+:::{tab-item} GPU
+```console
+python export_model.py text_generation --source_model Qwen/Qwen3-8B --weight-format int4 --config_file_path models/config.json --model_repository_path models --tools_model_type qwen3 --target_device GPU
+```
+```
+:::
+
+:::{tab-item} NPU
+```console
+python export_model.py text_generation --source_model Qwen/Qwen3-8B --config_file_path models/config.json --model_repository_path models --tools_model_type qwen3 --overwrite_models --target_device GPU
+```
+:::
+
+::::
+
+You can use similar commands for different models. Change the source_model and the tools_model_type out of phi4, llama2, quen3 and hermes3.
+> **Note:** The tuned chat template will be copied to the model folder as chat_template.jinja and the output parset will be set in the graph.pbtxt
+
+
+## Start OVMS
+
+Starting the model server is identical like with other demos with generative endpoints:
+**Deploying with Docker**
+
+Select deployment option depending on how you prepared models in the previous step.
+
+::::{tab-set}
+
+:::{tab-item} CPU
+
+Running this command starts the container with CPU only target device:
+```bash
+docker run -d --rm -p 8000:8000 -v $(pwd)/models:/models:ro openvino/model_server:2025.2 --rest_port 8000 --model_path /models/Qwen/Qwen3-8B --model_name Qwen/Qwen3-8B
+```
+:::
+
+:::{tab-item} GPU
+
+In case you want to use GPU device to run the generation, add extra docker parameters `--device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1)`
+to `docker run` command, use the image with GPU support. Export the models with precision matching the GPU capacity and adjust pipeline configuration.
+It can be applied using the commands below:
+```bash
+docker run -d --rm -p 8000:8000 --device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -v $(pwd)/models:/models:ro openvino/model_server:2025.2-gpu \
+--rest_port 8000 --model_path /models/Qwen/Qwen3-8B --model_name Qwen/Qwen3-8B
+```
+:::
+:::{tab-item} NPU
+
+Running this command starts the container with NPU enabled:
+```bash
+docker run -d --rm --device /dev/accel --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) \
+-p 8000:8000 -v $(pwd)/models:/models:rw openvino/model_server:latest-gpu --rest_port 8000 --model_path /models/Qwen/Qwen3-8B --model_name Qwen/Qwen3-8B
+```
+:::
+
+::::
+
+
+**Deploying on Bare Metal**
```

Comment:
I suppose we should mention somewhere below that only packages of OVMS with Python support have capabilities to run this demo